core/templates/directives/headroom.directive.ts

```diff
@@ -18,10 +18,6 @@
  * NB: Reusable component directives should go in the components/ folder.
  */
 
-/**
- * TODO(#12855): Remove angularjs headroom,
```

Comment:
Please update the PR title and description with Fix and fixes #12855

Reply:
done